Cargo.toml

```diff
@@ -54,12 +66,12 @@ hyper = "0.14"
 hyperlocal = "0.8"
 # close to same tokio version as dependent by hyper v0.14 and hyperlocal 0.8 -- things can go awry if they mismatch
 tokio = { version = "1", features = ["sync", "macros", "rt-multi-thread", "rt"] }
-opentelemetry = { version = "0.20.0", features = ["rt-tokio"] }
-tracing-opentelemetry = "0.21.0"
-opentelemetry-otlp = { version = "0.13.0", default-features = false, features = ["http-proto", "reqwest-client"] }
-tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
-opentelemetry-semantic-conventions = "0.12.0"
-tracing = { version = "0.1.40", features = ["async-await", "log"] }
+opentelemetry = { version = "0.20.0", features = ["rt-tokio"], optional = true }
+tracing-opentelemetry = {  version = "0.21.0", optional = true }
+opentelemetry-otlp = { version = "0.13.0", default-features = false, features = ["http-proto", "reqwest-client"], optional = true }
+tracing-subscriber = { version = "0.3.17", default-features = false, features = ["env-filter", "fmt"], optional = true }
+opentelemetry-semantic-conventions = { version = "0.12.0", optional = true }
+tracing = { version = "0.1.40", default-features = false, features = ["attributes"], optional = true }
```

Comment:
`tracing` should not be optional

Reply:
This could be only removed from tracing crate dependency. For others it resulted in errors like:
```
❯ cargo build --no-default-features --features otlp-tracing
error: failed to parse manifest at `/Users/mariuszreichert/workspace/github-projects/my/electrs/Cargo.toml`

Caused by:
  feature `otlp-tracing` includes `opentelemetry-otlp`, but `opentelemetry-otlp` is not an optional dependency
  A non-optional dependency of the same name is defined; consider adding `optional = true` to its definition.
```